lib/cisco_node_utils/radius_global.rb

```diff
@@ -137,5 +137,29 @@ def key_set(value, format)
                    key:   "#{value}")
       end
     end
+
+    # Get default source interface
+    def self.default_source_interface
+      config_get_default('radius_global', 'source_interface')
+    end
+
+    # Set source interface
+    def source_interface=(name)
+      if name
+        config_set(
+          'radius_global', 'source_interface',
+          state: '', source_interface: name)
+      else
+        config_set(
```

Comment:
Is your intent to use `default` as the manifest setting to remove the source_interface?

Reply:
On the puppet side we're looking for `'unset'`, which would then pass `nil` to node_utils.  For whatever reason that's what the netdev providers/utils have been doing.

Reply:
Interesting.  Currently the types (https://github.com/puppetlabs/netdev_stdlib/pull/25/files) only indicate that a String is accepted.  I think it would be clearer if we updated the input validation to indicate a string representing the interface or the string 1`'unset'` to remove the interface.  Thoughts?

Reply:
I'll have to see how things are being implemented with other vendor modules.